Tests/UtilityTests/ProgressBarTests.swift

```diff
@@ -12,46 +12,10 @@ import XCTest
 import Utility
 import SPMLibc
 @testable import Basic
+@testable import TestSupport
```

Comment:
We should make the class public instead of doing a testable import

Reply:
Fixed. Thanks!